datascience/tables.py

```diff
@@ -5678,12 +5678,46 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable view over the rows in a table.
+
+        args:
+        table- accepts a table object of class Table()
+        
+        >>> from datascience import Table
```

Comment:
`Table` should already be imported from other doctests above, no need to re-import it.